{
    "CVE-ID": "CVE-2012-2135",
    "NVD": {
        "Descriptions": "The utf-16 decoder in Python 3.1 through 3.3 does not update the aligned_end variable after calling the unicode_decode_call_errorhandler function, which allows remote attackers to obtain sensitive information (process memory) or cause a denial of service (memory corruption and crash) via unspecified vectors.",
        "Cvss2_BaseScore": 6.4,
        "Cvss3_BaseScore": 0,
        "Cvss2_VectorString": "AV:N/AC:L/Au:N/C:P/I:N/A:P",
        "Cvss3_VectorString": "",
        "CPEs": [
            {
                "URI": "cpe:/a:python:python:3.3:beta2",
                "FormattedString": "cpe:2.3:a:python:python:3.3:beta2:*:*:*:*:*:*",
                "VersionStartExcluding": "",
                "VersionStartIncluding": "",
                "VersionEndExcluding": "",
                "VersionEndIncluding": ""
            },
            {
                "URI": "cpe:/a:python:python:3.3",
                "FormattedString": "cpe:2.3:a:python:python:3.3:*:*:*:*:*:*:*",
                "VersionStartExcluding": "",
                "VersionStartIncluding": "",
                "VersionEndExcluding": "",
                "VersionEndIncluding": ""
            },
            {
                "URI": "cpe:/a:python:python:3.2:alpha",
                "FormattedString": "cpe:2.3:a:python:python:3.2:alpha:*:*:*:*:*:*",
                "VersionStartExcluding": "",
                "VersionStartIncluding": "",
                "VersionEndExcluding": "",
                "VersionEndIncluding": ""
            },
            {
                "URI": "cpe:/a:python:python:3.2.3",
                "FormattedString": "cpe:2.3:a:python:python:3.2.3:*:*:*:*:*:*:*",
                "VersionStartExcluding": "",
                "VersionStartIncluding": "",
                "VersionEndExcluding": "",
                "VersionEndIncluding": ""
            },
            {
                "URI": "cpe:/a:python:python:3.2.2150",
                "FormattedString": "cpe:2.3:a:python:python:3.2.2150:*:*:*:*:*:*:*",
                "VersionStartExcluding": "",
                "VersionStartIncluding": "",
                "VersionEndExcluding": "",
                "VersionEndIncluding": ""
            },
            {
                "URI": "cpe:/a:python:python:3.1.5",
                "FormattedString": "cpe:2.3:a:python:python:3.1.5:*:*:*:*:*:*:*",
                "VersionStartExcluding": "",
                "VersionStartIncluding": "",
                "VersionEndExcluding": "",
                "VersionEndIncluding": ""
            },
            {
                "URI": "cpe:/a:python:python:3.1.2",
                "FormattedString": "cpe:2.3:a:python:python:3.1.2:*:*:*:*:*:*:*",
                "VersionStartExcluding": "",
                "VersionStartIncluding": "",
                "VersionEndExcluding": "",
                "VersionEndIncluding": ""
            },
            {
                "URI": "cpe:/a:python:python:3.1.1",
                "FormattedString": "cpe:2.3:a:python:python:3.1.1:*:*:*:*:*:*:*",
                "VersionStartExcluding": "",
                "VersionStartIncluding": "",
                "VersionEndExcluding": "",
                "VersionEndIncluding": ""
            },
            {
                "URI": "cpe:/a:python:python:3.1",
                "FormattedString": "cpe:2.3:a:python:python:3.1:*:*:*:*:*:*:*",
                "VersionStartExcluding": "",
                "VersionStartIncluding": "",
                "VersionEndExcluding": "",
                "VersionEndIncluding": ""
            }
        ]
    },
    "Official": {
        "3.2.4rc1": {
            "source": [
                "https://github.com/python/cpython/blob/3.2/Misc/NEWS"
            ],
            "BTS": {
                "bpo14579": {
                    "ID": "bpo14579",
                    "url": "https://bugs.python.org/issue14579",
                    "date": "",
                    "nonce": "",
                    "section": ""
                }
            }
        },
        "2.7.4rc1": {
            "source": [
                "https://github.com/python/cpython/blob/v2.7.15rc1/Misc/NEWS.d/2.7.4rc1.rst"
            ],
            "BTS": {
                "bpo14579": {
                    "ID": "bpo14579",
                    "url": "https://bugs.python.org/issue14579",
                    "date": "8968",
                    "nonce": "Bcg1J1",
                    "section": "Core and Builtins"
                }
            }
        }
    }
}